pkg/pathutils/pathutils_test.go

```diff
@@ -0,0 +1,64 @@
+package pathutils
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"github.com/spf13/cobra"
+
+	infocmd "aspect.build/cli/cmd/aspect/info"
+	infopkg "aspect.build/cli/pkg/aspect/info"
+	"aspect.build/cli/pkg/ioutils"
+	"aspect.build/cli/pkg/pathutils"
+)
+
+func TestInvokeCmdInsideWorkspace(t *testing.T) {
```

Comment:
This was supposed to be a unit test, not an integration test. Performing disk IO operations with fixtures is not what we want in the `pkg` sub-packages.

To properly test the logic without calling `os.Stat` and `os.Getwd`, you will need to mock them.